packages/cpp-debug/src/browser/memory-widget/memory-table-widget.tsx

```diff
@@ -29,7 +29,7 @@ import { MemoryOptionsWidget } from './memory-options-widget';
 import { MemoryHoverRendererService, EasilyMappedObject } from '../utils/memory-hover-renderer';
 import { VariableDecoration, VariableFinder } from '../utils/memory-widget-variable-utils';
 import { MWMoreMemorySelect } from '../utils/memory-widget-components';
-import { MemoryProviderService } from '../memory-provider/memory-provider-service';
+import { hexStrToUnsignedLong } from '../../common/util';
```

Comment:
Looks like you accidentally removed the import for the memory provider service, so the build is failing.

Reply:
Oopsie, fixed! Thanks.